wagtail/admin/templates/wagtailadmin/pages/choose_parent.html

```diff
@@ -0,0 +1,31 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags%}
+
+{% block titletag %}Choose a parent page{% endblock %}
+
+{% block extra_css %}
+    {{ block.super }}
+    {{ form.media.css }}
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+    {% include "wagtailadmin/pages/_editor_js.html" %}
```

Comment:
```suggestion
```

This is probably no longer necessary now that we include the modal-workflow.js for the chooser is included everywhere. Could you recheck?